src/components/Shared/Form/Select.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC } from 'react';
+import styled from 'styled-components';
+import {
+  font_settings,
+  boxShadow_darkTheme_input,
+  boxShadow_lightTheme_input,
+  outline_focus,
+} from 'style/mixins';
+
+interface SelectProps {
+  options: string[];
+  onChange: (value: string) => void;
```

Comment:
czy label dla option jest taki sam jak jego value? 
np będziesz mieć owoce: 
gruszka, jabłko, brzoskwinia. 
to nie chcesz żeby inaczej były wyświetlane, a co innego szło do bazy?

Reply:
options: {value: string, label: string}[];